postgres/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM postgres:9.6
+FROM postgres:10.9
```

Comment:
@idgserpro Why not 10.14?! See https://hub.docker.com/_/postgres

Reply:
@avoinea only because it is the version we have in my company's infrastructure. Can we have several branches, with version 9?

Reply:
@avoinea  well, I think it could be Postgres 10.14. I will update.

Reply:
Done #3